platform/lib/utils/HeadDedupTransformer.js

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
The filename should start with a lowercase character

Reply:
uppercase when exporting a class